minikubek8s.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Install kubectl
+echo "----------------------------kubectl checking----------------------------"
+export kubectl=$(which kubectl)
+if [ -f "$kubectl" ]; then
+    echo "$kubectl already exist...................................."
+else
+    echo "kubectl does not exist...................................."
+    echo "Installing kubectl........................................"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    kubectl version
+fi
+
+# Install Brew
+echo "-----------------------------brew checking------------------------------"
+export brew=$(which brew)
+if [ -f "$brew" ]; then
+    echo "$brew already exist......................................."
+else
+    echo "brew does not exist......................................."
+    echo "Installing brew..........................................."
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
IMO, if the user does not have brew install, you should say: "brew not detected, please read requirements or info before proceeding. Please visit this this url". 
And may be the same thing for virtual box...